riocli/configtree/import_keys.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2024 Rapyuta Robotics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, Optional, Iterable
+from pathlib import Path
+
+import click
+from click_help_colors import HelpColorsCommand
+from benedict import benedict
+from yaspin.core import Yaspin
+
+from riocli.config import new_v2_client
+from riocli.configtree.etcd import import_in_etcd
+from riocli.configtree.revision import Revision
+from riocli.constants import Symbols, Colors
+from riocli.utils.spinner import with_spinner
+
+
+@click.command(
+    'import',
+    cls=HelpColorsCommand,
+    help_headers_color=Colors.YELLOW,
+    help_options_color=Colors.GREEN,
+)
+@click.option('--commit/--no-commit', 'commit', is_flag=True, type=bool,)
+@click.option('--update-head/--no-update-head', 'update_head', is_flag=True, type=bool)
+@click.option('--etcd-endpoint', 'etcd_endpoint', type=str, help='Import keys to local etcd instead of rapyuta.io cloud')
+@click.option('--etcd-port', 'etcd_port', type=int, help='Port for the etcd endpoint')
+@click.option('--etcd-prefix', 'etcd_prefix', type=str, help='Prefix to use for the key-space')
+@click.option('--project', 'with_project', is_flag=True, type=bool, help='Operate on the Trees in Project-scope.')
+@click.argument('tree-name', type=str)
+@click.argument('files', type=str, nargs=-1)
+@click.pass_context
+@with_spinner(text="Importing keys...")
+def import_keys(
+        _: click.Context,
+        tree_name: str,
+        files: Iterable[str],
+        commit: bool,
+        update_head: bool,
+        etcd_endpoint: Optional[str],
+        etcd_port: Optional[int],
+        etcd_prefix: Optional[str],
+        with_project: bool,
+        spinner: Yaspin,
+) -> None:
+    """
+    Imports Keys for the Config Tree from a YAML file.
```

Comment:
nit: Can we make sure that we don't mix a lot of title case words mid-sentences?
```suggestion
    Imports keys in a config tree from YAML files.
```

Reply:
Makes sense, I've fixed other instances but I guess better to keep Config Tree in title case as this is the resource. Let me know what you think.